functions.d/ambient_get_icomera_endpoint.fish

```diff
@@ -0,0 +1,9 @@
+function ambient_get_icomera_endpoint
```

Comment:
Hm, I think if this should maybe live in `icomera.fish` if it's not used anywhere else.